src/Store.ts

```diff
@@ -92,21 +94,24 @@ export const store = {
   },
 
   async initStore (): Promise<void> {
+    this.harbors = require("./assets/resources/ports.json")
     this.allData = await fetchOpsData()
     this.updateHistogramSlider()
     this.filterData(this.state.minDate, this.state.maxDate)
   },
 
   displayMap (): void {
-    this.state.baseMap.display(this.state.timeFilteredData)
+    this.state.baseMap.init()
   },
 
   updateBasemap (index: number): void {
     this.state.baseMap.setCurrentBasemap(index)
   },
 
   updateMap (): void {
-    this.state.baseMap.update(this.state.timeFilteredData)
+    this.state.baseMap.removeMarkers()
+    this.state.baseMap.addHarbors(this.harbors)
```

Comment:
Harbors are removed and added again when dates are updated whereas they are independent from the dates

Reply:
Yep ! I would love to know how to create a layer for harbors, but I don't how to do it with MapBoxGlJS. Any idea ?

Reply:
I found this : https://docs.mapbox.com/mapbox.js/example/v1.0.0/non-exclusive-markers/

Reply:
In BaseMap.ts, we store all markers in `this.markers`, but we can store the harbors in another variable to manager them.